requirements/edx/development.txt

```diff
@@ -68,7 +72,7 @@ astroid==2.8.4
     #   -r requirements/edx/testing.txt
     #   pylint
     #   pylint-celery
-async-timeout==3.0.1
+async-timeout==4.0.0
```

Comment:
No breaking change mentioned in the major version release notes: https://github.com/aio-libs/async-timeout/blob/master/CHANGES.rst#400-2021-11-01